Python_Engine/Python_Engine.csproj

```diff
@@ -70,4 +77,8 @@
     <Folder Include="Resources\" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>xcopy "$(TargetDir)$(TargetFileName)"  "C:\\ProgramData\\BHoM\\Assemblies" /Y
```

Comment:
Not sure what's the syntax in the post-build events, but also here, maybe useful to use env variables?
"%ALLUSERSPROFILE%\\BHoM\\Assemblies" or "$(ALLUSERSPROFILE)\\BHoM\\Assemblies"

Reply:
The post build is currently set to the BHoM compliant method which is used by all of the other toolkits - see https://github.com/BHoM/admin/issues/5 :smile:

Reply:
Oh, right, sorry!